python/example_code/emr/pypi.py

```diff
@@ -0,0 +1,64 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
You might want to choose a different name for this file if it's not already well established. Pypi is the Python Package Index that hosts the python software accessed through pip.

Reply:
Heh, good point, that hadn't occurred to me. I changed it to pyspark_estimate_pi.py.